src/main/java/com/team4/testingsystem/services/impl/RestrictionsServiceImpl.java

```diff
@@ -35,4 +38,29 @@ public void checkTestContainsQuestion(Test test, Question question) {
                 + question.getId());
         }
     }
+
+
+    @Override
+    public void checkGradeIsCorrect(int grade) {
+        if (grade < 0 || grade > 10) {
+            throw new IllegalGradeException();
+        }
+    }
+
+    @Override
+    public void checkModule(Question question) {
+        if (!question.getModule().getName().equals(Modules.ESSAY.getName())
+            && !question.getModule().getName().equals(Modules.SPEAKING.getName())) {
```

Comment:
Maybe let's replace it with smth like `checkModule(Question, Modules)` or `checkModuleIn(Question, Modules[])`?
Or, at least, change name `checkModule`...